install.sh

```diff
@@ -1,25 +1,51 @@
+#!/bin/bash
+
+###############################################################################
+# Instalation Script for Gitshots
+# Repository: https://github.com/ranman/gitshots-client
+# gitshots.com - Take a picture and collect some stats every time you commit!
+###############################################################################
+
+USERHOME=$(dscl . -read /Users/`whoami` NFSHomeDirectory | awk -F": " '{print $2}')
+POST_COMMIT_FILE=${USERHOME}/bin/post-commit.py
+GIT_TEMPLATE_DIR=${USERHOME}/.git_template
+GIT_TEMPLATE_HOOKS_DIR=${GIT_TEMPLATE_DIR}/hooks
+
+printf "\nPutting post-commit.py in bin.\n"
+if [[ -f $POST_COMMIT_FILE ]]; then
+   printf "\nRemoving old file: $POST_COMMIT_FILE\n"
+   rm $POST_COMMIT_FILE
+else
+	mkdir -p ~/bin
+fi
+
+if [[ $1 && -f $1 ]]; then
+   	printf "\nSymbolic link from: $1 to $POST_COMMIT_FILE\n"
+	ln -s $1 $POST_COMMIT_FILE
+else
+	printf "\nDownload from github master branch: $POST_COMMIT_FILE\n"
+	curl https://raw.githubusercontent.com/ranman/gitshots-client/master/post-commit.py > $POST_COMMIT_FILE
+fi
+
+printf "\nCreating git_template dir.\n"
+mkdir -p $GIT_TEMPLATE_HOOKS_DIR
+git config --global init.templatedir $GIT_TEMPLATE_DIR
```

Comment:
... so, because of this one, I want to close this pull request without the merge.
I want to propose to make gitshots more flexible accordantly  with the user preference by project.
Idea for commands:
* gitshots init   > (same as git init but with our super post-commit hook) 
* gitshots init --no-upload --no-location > (gitshots will take the shot, but it will be only available in its local folder and it will not fetch your location
* gitshots config --no-upload --no-location > (Changes the config within a project with .git)
* gitshots config --global --no-upload --no-location > (Changes the config globally )

(Copy & Past to )